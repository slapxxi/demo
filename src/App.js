// @flow
import React, { Component } from 'react';
import glamorous from 'glamorous';

type Props = {};

class App extends Component<Props> {
  render() {
    return <Container>Welcome to React</Container>;
  }
}

const Container = glamorous.div({});

export default App;
