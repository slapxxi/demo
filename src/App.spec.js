import React from 'react';
import { render } from 'enzyme';
import App from './App';

it('renders without crashing', () => {
  const component = render(<App />);
  expect(component).toMatchSnapshot();
});
