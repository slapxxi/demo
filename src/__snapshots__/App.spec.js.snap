// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  class="css-nil"
>
  Welcome to React
</div>
`;
